dev-packages/application-manager/src/application-package-manager.ts

```diff
@@ -89,9 +86,7 @@ export class ApplicationPackageManager {
             const inspectArg = mainArgs.splice(inspectIndex, 1)[0];
             options.execArgv = ['--nolazy', inspectArg];
         }
-        return this.__process.bunyan(
-            this.__process.fork(this.pck.backend('main.js'), mainArgs, options)
-        );
+        this.__process.fork(this.pck.backend('main.js'), mainArgs, options);
```

Comment:
`return` or `await`?

Reply:
OK, but then I do not understand this :) At least partially. What would be the difference?

 1. I cannot do `await`. `fork` is the `cp.fork` and `spawnBin` is just a `cp.spawn`. Why do we need to wait for anything here?
 2. It does not make any sense to return with the `cp.ChildProcess`, does it?
 3. I could write a `return` statement but that is equivalent with the current code.

Reply:
ok, it used to be promisified, if `theia start` keeps running till started processes are alive it and interrupting it kills processes then i am fine